runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -151,4 +151,25 @@ public Query query(@Nonnull final String namespace) {
                 this.runtime.getObjectsView(),
                 namespace);
     }
+
+    /**
+     * Subscribe to a specific a table in a namespace or the entire namespace.
+     * Objects returned will honor transactional boundaries
+     * @param streamListener - callback context.
+     * @param namespace - the CorfuStore namespace to subscribe to.
+     * @param tableName - If specified, only updates from this table will be returned.
+     * @param timestamp - If specified, all stream updates from this timestamp will be returned
```

Comment:
Did not understand the timestamp doc.

Reply:
The usage is 
1. take a snapshot which returns a timestamp
2. process all table entries at that timestamp
3. start streaming from that timestamp to process all remaining updates that happened after the snapshot was taken
perhaps I can put the above in the comment?

Reply:
Yes please add it as comment.